AdamJacobs/dojo_survey/server.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request, redirect
+app = Flask(__name__)
+
+name = None
+location = None
+@app.route('/index.html')
+def index():
+ 	return render_template('index.html')
+
+@app.route('/students', methods = ['POST'])
+def create_student():
+  if request.method == 'POST':
+  	name = request.form['name']
+  	location = request.form['location']
+  	language = request.form['language']
+  	if request.form['comment']:
+  		comment = request.form['comment']
+  	else:
+  		comment = None
+  	print name
+  	print location
+  	print language
+  	if request.form['comment']:
```

Comment:
may break if key comment does not exist in the request.form dictionary. Find a way to safely check for keys